src/Pages/signin/Components/SignInForm.tsx

```diff
@@ -1,22 +1,80 @@
 import React from "react";
-import Box from "@material-ui/core/Box";
-import Typography from "@material-ui/core/Typography";
+import styled from "styled-components";
+import Background from "../../../assets/StrokeBackground.png";
+import LoginLogo from "../../../assets/LoginLogo.png";
+//React router
+import {Link} from 'react-router-dom';
+//color
```

Comment:
All of the components are not need to export (except SignInForm) after you complete the below task.